magprime/templates/emails/second_shirt.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<head></head>
+<body>
+
+<p>Hello {{ attendee.first_name }},</p>
+
+<p>
+    You are receiving this e-mail because you're a staffer eligible for complimentary shirts.
```

Comment:
You've spelled it "complementary" in other parts of the code. I, uh, actually don't know which is the correct spelling. 😅